src/org/zaproxy/zap/extension/websocket/files/scripts/templates/websocketpassive/Passive WebSocket Scan Template.js

```diff
@@ -0,0 +1,81 @@
+// Passive scan rules should not send messages
+// Right click the script in the Scripts tree and select "enable"  or "disable"
+
+var PluginPassiveScanner = Java.type("org.zaproxy.zap.extension.websocket.pscan.WebSocketPluginPassiveScanner");
+
+OPCODE_CONTINUATION = 0x0;
+
+OPCODE_TEXT = 0x1;
+OPCODE_BINARY = 0x2;
+
+OPCODE_CLOSE = 0x8;
+OPCODE_PING = 0x9;
+OPCODE_PONG = 0xA;
+
+RISK_INFO 	= 0;
+RISK_LOW 	= 1;
+RISK_MEDIUM = 2;
+RISK_HIGH 	= 3;
+
+CONFIDENCE_LOW = 1;
+CONFIDENCE_MEDIUM = 2;
+CONFIDENCE_HIGH = 3;
+
+
+/**
+ * This function is Optional. By default allows all the message to pass at the scan phase.
+ * Return {@code true} to pass WebSocket message to scan function.
+ * @param msg WebSocket message (WebSocketMessage DTO)
+ */
+function applyScanToMessage(msg) {
+    if(msg.opcode == OPCODE_TEXT && !msg.isOutgoing){
+        return true;
+    }
+    return false;
+}
+
+/**
+ * Passively scans an WebSocket message. The scan function will be called for
+ * messages via ZAP. Script scans only the messages which is passed from
+ * from WebSocketPassiveScript#applyScanToMessage (defined bellow)
+ *
+ * @param alrt - the WebSocketPassiveScanAlert interface provides the raiseAlert method in order
```

Comment:
Maybe this should be `helper` similar to other scripts/functionality?

Reply:
Right, so as to be more consistent.  websocket/alerts/WebSocketPassiveScanAlert class name should be changed as well?